operations/accounts.md

```diff
@@ -0,0 +1,64 @@
+# Accounts
+
+## Merge accounts
+
+Merges ore or more accounts to another.
```

Comment:
```suggestion
Merges two or more accounts together. These must either be all Company accounts or all Customer accounts.
```

Reply:
I think we should also explain what will be the ID of the merged accounts. Will it be a new record with a new ID, or will it be one of the IDs of the merged accounts? Do we also need to say anything about the merging rules and possible conflicts?

Reply:
**Addendum:** I just realised that this is the intention of SourceIds and TargetId. So the operation merges the source records into the target record and so the record with TargetId will be the new, merged record. Is that correct? Ok, so we should say that in the description. Something like... "The given source accounts will be merged into the given target account and the merged account will keep the target account ID."

Reply:
Exactly, `SourceIds` will be deleted and merged into the `TargetId` which keeps it's id. I've add "Merges two or more accounts together." also, because what you have suggested for me is like description.

Is it ok like that now?